src/citrine/informatics/workflows/analysis_workflow.py

```diff
@@ -0,0 +1,75 @@
+from typing import List, Optional, Union
+from uuid import UUID
+
+from citrine._rest.engine_resource import EngineResourceWithoutStatus
+from citrine._rest.resource import Resource
+from citrine._serialization import properties
+from citrine.informatics.workflows.workflow import Workflow
+from citrine.gemd_queries.gemd_query import GemdQuery
+
+
+class LatestBuild(Resource['LatestBuild']):
+    """Info on the latest analysis workflow build."""
+
+    status = properties.Optional(properties.String, 'status', serializable=False)
+    failures = properties.List(properties.String, 'failure_reason', default=[], serializable=False)
+    query = properties.Optional(properties.Object(GemdQuery), 'query', serializable=False)
+
+
+class AnalysisWorkflow(EngineResourceWithoutStatus['AnalysisWorkflow'], Workflow):
+    """An analysis workflow stored on the platform.
+
+    Note that plots are not fully supported. They're captured as raw JSON in order to facilitate
+    cloning an existing workflow, but no facilities are provided to validate them in the client.
+    """
+
+    uid = properties.UUID('id', serializable=False)
+    name = properties.String('data.name')
+    description = properties.String('data.description')
+    snapshot_id = properties.Optional(properties.UUID, 'data.snapshot_id')
+    plots = properties.List(properties.Raw, 'data.plots', default=[])
```

Comment:
Since we are explicitly not supporting plot objects, wouldn't it make more sense to make this a private variable?  Otherwise, if we ever did add Plot objects, we'd have a breaking change going from a dict to a class.